### ISSUE TITLE
Hi! I want to use this code. But I think my version is not correct.

### ISSUE THREAD

**Issue by zeldabean** [leggedrobotics/darknet_ros#145]
What version do you use??

Plez give me your advice. 

Now, I’m using ros kinetic, Cuda 9.0, cuDnn 7.1.4, tensorflow_gpu 1.12.0, opencv 3.4.0(cmake 3.12.2)

Plez help me T.T

**Comment by zeldabean** [comment]
Yes, but It doesn't work. I think opencv version is not suitable. I saw some errors about opencv funtion.

**Comment by Chrislzy1993** [comment]
@zeldabean , https://github.com/pjreddie/darknet/issues/918

**Comment by zeldabean** [comment]
thanks!! : )
